resources/assets/sass/app.scss

```diff
@@ -134,3 +134,14 @@ div:empty {
     @apply .font-bold;
   }
 }
+
+//
+// Login / SignUp
+//
+@responsive {
+  .opencfp-login,
+  .opencfp-signup {
```

Comment:
Extracting two different classes here, as we might want to address these forms separately at some point in time.